schemas/yazi.json

```diff
@@ -23,18 +23,29 @@
 				"sort_by": {
 					"enum": [
 						"none",
-						"alphabetical",
-						"created",
 						"modified",
+						"created",
+						"extention",
+						"alphabetical",
 						"natural",
-						"size"
+						"size",
+						"random"
 					]
 				},
 				"sort_sensitive": { "type": "boolean" },
 				"sort_reverse": { "type": "boolean" },
 				"sort_dir_first": { "type": "boolean" },
 				"sort_translit": { "type": "boolean" },
-				"linemode": { "enum": ["none", "size", "permissions", "mtime"] },
+				"linemode": {
+					"enum": [
+						"none",
+						"size",
+						"ctime",
+						"mtime",
+						"permissions",
+						"owner"
+					]
+				},
```

Comment:
This actually is *not* an enum. It can be any string, since these can be custom defined. I'm not sure if doing a union between an enum and a string would give useful autocomplete but maybe it is worth a try?

Reply:
this works as expected
```json
				"linemode": {
					"anyOf": [
						{ "type": "string", "maxLength": 20,  },
						{
							"enum": [
								"none",
								"size",
								"ctime",
								"mtime",
								"permissions",
								"owner"
							]
						}
					]
				},
```

Reply:
## when string length is zero
![zero](https://github.com/user-attachments/assets/aedcef3d-3977-410a-986e-15385e987882)

## when string length is 20+
![twenty](https://github.com/user-attachments/assets/22ce1b15-b396-44f4-bef0-b476d57a620c)

## autocomplete
![autocomplete](https://github.com/user-attachments/assets/e575de31-56d5-4efa-8af3-cd25af31cca5)

Reply:
This looks nice, TIL about that

Reply:
Learn about what? 😅 The autocomplete?

Reply:
Yep!

Reply:
Yeah that's mostly the reason I made these lol, along with validation ofc

Reply:
For some reason, my autocomplete has never worked for me, only validation worked, I thought it was unsupported haha - need to find some time to fix it

![screenshot-002028](https://github.com/user-attachments/assets/756b9850-df73-46f5-9617-8a7eaa98c771)

Reply:
Huh weird! Make sure you have the Taplo (or some other) LSP running.

Reply:
Oh shoot that's the reason! I can't remember why I [commented it out](https://github.com/sxyazi/dotfiles/blob/c5bf2398b141810e04b7ce07447144518dad143b/nvim/lua/plugins/lsp.lua#L333). I've been using ESLint to validate the scheme💀😕

Reply:
Sweet! Glad it's working now, and that you can find it useful 😁